Library/SharedFunctionsTests.swift

```diff
@@ -160,4 +161,14 @@ final class SharedFunctionsTests: TestCase {
     minuteTest.assertValues(["00", "00", "00"])
     secondTest.assertValues(["02", "01", "00"])
   }
+
+  func testOnePasswordButtonIsHidden() {
+    if #available(iOS 12, *) {
```

Comment:
Making the tests version independent

Reply:
I realize that...I'm not sure how I feel about this because if we don't make this explicit in our test suite we're essentially giving away a lot of context on why some tests only test one code path. Have you dealt with this before?

Reply:
Well, it's just that we know our CI is never going to run this on anything but the latest iOS 🤔 We really only do these checks in application code as like a graceful fallback.

Either way the code is redundant here.

I suppose one could leave a comment or somehow inject the iOS 12 dependency, what if the func looked like this:

```swift
internal func is1PasswordButtonHidden(_ isHidden: Bool, isIOS12Available: Bool = isIOS12Available()) -> Bool {
  guard isIOS12Available else { return isHidden }

  return true
}

private func isIOS12Available() -> Bool {
  guard #available(iOS 12, *) else { return false }

  return true
}
```

Our test could then inject the iOS 12 availability:

```swift
func testOnePasswordButtonIsHidden() {
  XCTAssertTrue(is1PasswordButtonHidden(true, isIOS12Available: true))
  XCTAssertTrue(is1PasswordButtonHidden(true, isIOS12Available: true))
  XCTAssertTrue(is1PasswordButtonHidden(true, isIOS12Available: false))
  XCTAssertTrue(is1PasswordButtonHidden(true, isIOS12Available: false))
}
```